plotly-plot-behavior.js

```diff
@@ -204,16 +204,20 @@
         return self.fire('plotly-relayout', {data: data});
       };
 
+      self._addListenerIfNotExists = function(plotDiv, type, listener){
```

Comment:
this formatting is all wrong; also, you're accessing unexposed API points—why is this code necessary?